### ISSUE TITLE
Modal: Help icon missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#849]
### Expected behavior

Help icon has text equivalent that can be read by assistive technology. 
### Actual behavior

No text equivalent available.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.9.0](https://github.com/blackbaud/skyux/releases/tag/v1.9.0)
